pandas/tests/sparse/test_frame.py

```diff
@@ -1118,6 +1120,28 @@ def test_isnotnull(self):
                             'B': [True, False, True, True, False]})
         tm.assert_frame_equal(res.to_dense(), exp)
 
+    @pytest.mark.parameterize('index', [None, list('abc')])
```

Comment:
no, what you need to do is create this as a function (and not put it in the class). parameterization doesn't work if:
``class Foo(tm.TestCase)`` which is how we start all of the classes. So easiest to make a stand alone function. You can change it to ``class Foo(object)``, but then all of the ``self.assert*`` methods have to be changed to ``pytest`` style. 

I have done that in a few cases.